py/kubeflow/kubeflow/cd/jwa.py

```diff
@@ -0,0 +1,48 @@
+""""Argo Workflow for building Jupyter web app's OCI image using Kaniko"""
+import os
+
+from kubeflow.kubeflow import ci
+from kubeflow.testing import argo_build_util
+
+TAG = os.getenv("PULL_BASE_SHA", "kaniko-local-test")
```

Comment:
I think a better naming scheme would be: <branch-name>-<commit-id>. The commit ID is the first 7 characters of the SHA1 hash.

Reply:
Also, by using the branch-name, fixes cherry picked fixes for specific Kubeflow release version can easily be distinguished from the containers built from the master branch.

Reply:
> The commit ID is the first 7 characters of the SHA1 hash.

Could you elaborate on this one? AFAIK the commit ID is the SHA1 of the important parts of the commit (date, log message etc). Could you point me to some documentation about the 7 chars?

> I think a better naming scheme would be:
> Also, by using the branch-name, fixes cherry picked fixes for specific Kubeflow release version can easily be distinguished from the containers built from the master branch.

Yeap, let's go with `${PULL_BASE_REF}-${PULL_BASE_SHA}[0:7]`. This was also the case with the previous images https://console.cloud.google.com/gcr/images/kubeflow-images-public/GLOBAL/jupyter-web-app?gcrImageListsize=30

Reply:
In the end I had to do `base-commit[0:8]` in order for the parsed length to be the same with what it was for the previous images.

@DavidSpek the images now have described format. Should I go on and resolve this conversation?

Reply:
@kimwnasptd Does the parsed length matter for something? 
Documentation on Git Commit: https://stackoverflow.com/questions/43763896/whats-the-difference-between-commit-id-and-sha1-hash-in-git

Reply:
Can't think of something, we should be good to go

Reply:
Alright, this can be resolved. Please see my comment below though.